README.md

```diff
@@ -2,20 +2,40 @@
 
 Discord BOT
 
-[Invite it to your server](https://discord.com/api/oauth2/authorize?client_id=913903801033981952&permissions=2147560448&scope=bot)
+- [PROD bot invite](https://discord.com/api/oauth2/authorize?client_id=913903801033981952&permissions=2147560448&scope=bot)
+- [TEST bot invite](https://discord.com/api/oauth2/authorize?client_id=976075945293266984&permissions=2147560448&scope=bot)
 
 ## Required env vars
 
 **BOT_TOKEN**: Discord bot token
 
 ## Running
 
+### With Node
+
 `yarn start`
 
-Alternatively, to test it will not fail on Heroku, install `heroku` cli and run:
+### With Docker
+
+```bash
+$ docker build . -t robot-cow-bot
```

Comment:
nit: why the $. Can we delete it so its easier to copy-paste?

Reply:
To indicate it's a non-root user, common in bash snippets.
Will remove